source/iface/integer-functions.rst

```diff
@@ -7,3 +7,855 @@
 *****************
 Integer functions
 *****************
+
+The function descriptions in this section use the term
+generic integer type to represent the following types:
+
+``char``
+
+``signed char``
+
+``short``
+
+``int``
+
+``long``
+
+``long long``
+
+``unsigned char``
+
+``unsigned short``
+
+``unsigned int``
+
+``unsigned long``
+
+``unsigned long long``
+
+``marray<char, N>``
+
+``marray<signed char, N>``
+
+``marray<short, N>``
+
+``marray<int, N>``
+
+``marray<long, N>``
+
+``marray<long long, N>``
+
+``marray<unsigned char, N>``
+
+``marray<unsigned short, N>``
+
+``marray<unsigned int, N>``
+
+``marray<unsigned long, N>``
+
+``marray<unsigned long long, N>``
+
+``vec<int8_t, N>``
+
+``vec<int16_t, N>``
+
+``vec<int32_t, N>``
+
+``vec<int64_t, N>``
+
+``vec<uint8_t, N>``
+
+``vec<uint16_t, N>``
+
+``vec<uint32_t, N>``
+
+``vec<uint64_t, N>``
+
+``__swizzled_vec__`` that is convertible to ``vec<int8_t, N>``
+
+``__swizzled_vec__`` that is convertible to ``vec<int16_t, N>``
+
+``__swizzled_vec__`` that is convertible to ``vec<int32_t, N>``
+
+``__swizzled_vec__`` that is convertible to ``vec<int64_t, N>``
+
+``__swizzled_vec__`` that is convertible to ``vec<uint8_t, N>``
+
+``__swizzled_vec__`` that is convertible to ``vec<uint16_t, N>``
+
+``__swizzled_vec__`` that is convertible to ``vec<uint32_t, N>``
+
+``__swizzled_vec__`` that is convertible to ``vec<uint64_t, N>``
```

Comment:
Can this be turned into a bullet item list so that it takes up space?  I think the common function section already does that, for example.  In the current form, this list takes up more than a full screen height, with at least half of that being spacing between the lines.